setup.py

```diff
@@ -6,8 +6,10 @@
 
 import os.path
 import sys
-
+import glob
 import setuptools
+from setuptools.command.develop import develop
```

Comment:
Maybe make defaultness explicit as `install` and `develop` are too ambiguous:
```suggestion
from setuptools.command.develop import develop as DefaultDevelopCommand
```